domains/certificates/Query.API/API/KeyIssuer/Startup.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Marten;
+using Marten.Schema;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace API.KeyIssuer;
+
+public static class Startup
+{
+    public static void AddKeyIssuerService(this IServiceCollection services) =>
+        services.ConfigureMarten(o =>
+        {
+            o.Schema
+                .For<KeyIssuingDocument>()
+                .UniqueIndex(UniqueIndexType.Computed, "uidx_meteringpointowner");
```

Comment:
There is no identity defined for KeyIssuingDocument, but it should most likely be "MeteringPointOwner". There are different ways of setting the identity if not naming a property "Id": https://martendb.io/documents/identity.html#overriding-the-choice-of-id-property-field